server/api.go

```diff
@@ -50,12 +49,12 @@ type dexAPI struct {
 }
 
 type vehicleIDTokenClaims struct {
-	Issuer   string   `json:"iss"`
-	Subject  string   `json:"sub"`
-	Audience audience `json:"aud,omitempty"`
-	Expiry   int64    `json:"exp"`
-	IssuedAt int64    `json:"iat"`
-	UserID   string   `json:"userId"`
+	Issuer         string   `json:"iss"`
+	Subject        string   `json:"sub"`
+	Audience       audience `json:"aud,omitempty"`
+	Expiry         int64    `json:"exp"`
+	IssuedAt       int64    `json:"iat"`
+	UserEthAddress string   `json:"userEthAddress"`
```

Comment:
Looking at the [claims registry](https://www.iana.org/assignments/jwt/jwt.xhtml) it seems to me that `snake_case` is the norm. `user_eth_address`?